.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "\U0001F4A1 Feature request"
+about: "Suggest an idea for this project \U0001F914"
+title: ''
+labels: Feature request
+assignees: ''
+
+---
+
+**Context**
```

Comment:
Nitpick: will it be better if we have the same font size for the subtitles in all these three files?

Reply:
Sure thing, changed all the headers to h4 to be consistent